game-core/src/main/java/games/strategy/engine/lobby/common/IRemoteHostUtils.java

```diff
@@ -18,4 +26,12 @@
   String shutDownHeadlessHostBot(String hashedPassword, String salt);
 
   String getSalt();
+
+  static RemoteName newRemoteNameForNode(final INode node) {
+    checkNotNull(node);
+
+    return new RemoteName(
+        "games.strategy.engine.lobby.server.RemoteHostUtils:" + node.toString(),
```

Comment:
I'd consider this kinda cheated.
The interface now sort of holds an indirect backreference to the implementing type which is meh.

Reply:
Yeah, it looks weird, but it's really just an opaque string.  There's no reflection or anything that ties it literally to `RemoteHostUtils`.  We can always change these channel names in the next incompatible release if they are confusing.  We just need an agreed-upon-beforehand name both client and server can use to connect to each other.